lib/nerves_network/eapol_manager.ex

```diff
@@ -352,10 +341,13 @@ defmodule Nerves.Network.EAPoLManager do
     {:noreply, s}
   end
 
-  # Will be called on wpa_supplicant's port's exit
+  # Will be called on wpa_supplicant's port's exit. My come asynchronously so if stop is being invoked
```

Comment:
consider adding some commas and rephrasing as currently hard to understand that line